libs/btp/navigation/navigation.component.ts

```diff
@@ -94,6 +95,12 @@ export class NavigationComponent
         this.type.set(type);
     }
 
+    /** @hidden */
+    @Input({ alias: 'homeSeparator', transform: (value: any) => coerceBooleanProperty(value) })
+    set _homeSeparator(value: boolean) {
+        this.homeSeparator.set(value);
+    }
```

Comment:
Use `BooleanInput` instead, otherwise it will not work with the string values like `"" | "true"` and etc